File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/fragment/help/HelpFragment.java
package com.zuk0.gaijinsmash.riderz.ui.fragment.help;

import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;
import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.zuk0.gaijinsmash.riderz.R;

public class HelpFragment extends Fragment {

    private View mInflatedView;

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        mInflatedView = inflater.inflate(R.layout.view_help, container, false);
        return mInflatedView;
    }

    @Override
    public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
        Button mButton = mInflatedView.findViewById(R.id.help_report_button);
        mButton.setOnClickListener(v -> {
            Intent intent = new Intent(Intent.ACTION_SEND);
            intent.setType("message/rfc822")
                    .putExtra(Intent.EXTRA_EMAIL, new String[] {"zuk0.hack@gmail.com"})
                    .putExtra(Intent.EXTRA_SUBJECT, "***RIDERZ BUG REPORT***")
                    .putExtra(Intent.EXTRA_TEXT, getResources().getString(R.string.help_intent_text));
            startActivity(Intent.createChooser(intent, getResources().getString(R.string.help_intent_title)));
        });
    }

    @Override
    public void onActivityCreated(Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        setRetainInstance(true);
    }
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/help/HelpFragment.kt
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.help

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import com.google.android.material.appbar.AppBarLayout
import com.zuk0.gaijinsmash.riderz.R
import com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.BaseFragment

class HelpFragment : BaseFragment() {
    private var mInflatedView: View? = null
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        mInflatedView = inflater.inflate(R.layout.fragment_help, container, false)
        return mInflatedView
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        val mButton = mInflatedView?.findViewById<Button>(R.id.help_report_button)
        mButton?.setOnClickListener { v: View? ->
            val intent = Intent(Intent.ACTION_SEND)
            intent.setType("message/rfc822")
                    .putExtra(Intent.EXTRA_EMAIL, arrayOf("zuk0.hack@gmail.com"))
                    .putExtra(Intent.EXTRA_SUBJECT, "***RIDERZ BUG REPORT***")
                    .putExtra(Intent.EXTRA_TEXT, resources.getString(R.string.help_intent_text))
            startActivity(Intent.createChooser(intent, resources.getString(R.string.help_intent_title)))
        }
        super.collapseAppBar(activity)
        super.setTitle(activity, getString(R.string.help_title))
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        retainInstance = true
    }
}